src/api/app/models/user.rb

```diff
@@ -332,7 +332,7 @@ def self.realname_for_login(login)
   # state an password hash type being in the range of allowed values.
   def validate
     # validate state and password has type to be in the valid range of values
-    errors.add(:password_hash_type, 'must be in the list of hash types.') unless User.password_hash_types.include? password_hash_type
+    errors.add(:password_hash_type, 'must be in the list of hash types.') unless password_hash_type.in?(User.password_hash_types.include?)
```

Comment:
`password_hash_type.in?(User.password_hash_types)` you left the `include?` :rofl: